FirebaseRemoteConfig/Sources/RCNConfigSettings.m

```diff
@@ -102,6 +102,14 @@ - (instancetype)initWithDatabaseManager:(RCNConfigDBManager *)manager
     _userDefaultsManager = [[RCNUserDefaultsManager alloc] initWithAppName:appName
                                                                   bundleID:_bundleIdentifier
                                                                  namespace:_FIRNamespace];
+
+    // Check if the config database is new. If so, clear the configs saved in userDefaults.
+    if ([_DBManager isNewDatabase]) {
```

Comment:
How can this test ever fail? I don't see gIsNewDatabase ever set to NO.

Reply:
The test case setup method above creates a new test database. We just want to verify that this flag is set to true whenever an existing database is not found.